layouts/partials/head.html

```diff
@@ -1,46 +1,87 @@
 <head>
-    <!-- Meta tags -->
-    <meta charset="UTF-8">
-    <meta name="description" content="{{if .IsHome}}{{$.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}">
-    <title>{{ if .Title }}{{ .Title }}{{ else }}{{ $.Site.Data.config.page_title }}{{ end }}</title>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="shortcut icon" type="image/png" href="{{$.Site.BaseURL}}/icon.png" />
+  <!-- Meta tags -->
+  <meta charset="UTF-8" />
+  <meta
+    name="description"
+    content="{{if .IsHome}}{{$.Site.Data.config.description}}{{else}}{{.Summary}}{{end}}"
+  />
+  <title>
+    {{ if .Title }}{{ .Title }}{{ else }}{{ $.Site.Data.config.page_title }}{{
+    end }}
+  </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link
+    rel="shortcut icon"
+    type="image/png"
+    href="{{$.Site.BaseURL}}/icon.png"
+  />
 
-    <!-- CSS Stylesheets and Fonts -->
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Source+Sans+Pro:wght@400;600;700&family=Fira+Code:wght@400;700&display=swap" rel="stylesheet">
-    {{$sass := resources.Match "styles/[!_]*.scss" }}
-    {{$css := slice }}
-    {{range $sass}}
-    {{$scss := . | resources.ToCSS (dict "outputStyle" "compressed") }}
-    {{$css = $css | append $scss}}
-    {{end}}
-    {{$finalCss := $css | resources.Concat "styles.css" | resources.Fingerprint "md5" | resources.Minify  }}
-    <link href="{{$finalCss.Permalink}}" rel="stylesheet">
+  <!-- CSS Stylesheets and Fonts -->
+  <link
+    href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&family=Source+Sans+Pro:wght@400;600;700&family=Fira+Code:wght@400;700&display=swap"
+    rel="stylesheet"
+  />
+  {{$sass := resources.Match "styles/[!_]*.scss" }} {{$css := slice }} {{range
```

Comment:
can you re-expand these into separate lines?

Reply:
Added!